### ISSUE TITLE
Version for 1.4.x

### ISSUE THREAD

**Issue by Portugao** [zikula-modules/Reviews#39]

**Comment by AlexL777** [comment]
Is this module already suitable for Zikula 1.4.6 or do you need to make adjustments for compatibility? I would like to install the module and also test.

**Comment by Portugao** [comment]
No, I don't think so. Would be fine if you could test...

**Comment by AlexL777** [comment]
No, it shows me that reviews are not yet compatible. Since I have to wait until there is a next module version.
